db/migrations/0003_alter_order_created_at.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.0.2 on 2024-11-21 18:18
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_user_order_alter_movie_actors_alter_movie_genres_and_more'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='order',
+            name='created_at',
+            field=models.DateTimeField(default=django.utils.timezone.now),

```

Comment:
Consider whether using `auto_now_add=True` might be more appropriate for the `created_at` field if it is intended to be set automatically when an `Order` instance is created. Using `default=django.utils.timezone.now` allows for manual setting of the field, which might not be desired.